src/AElf.Runtime.WebAssembly/WebAssemblySmartContractRunner.cs

```diff
@@ -21,9 +21,17 @@ public WebAssemblySmartContractRunner(IExternalEnvironment externalEnvironment)
 
     public async Task<IExecutive> RunAsync(SmartContractRegistration reg)
     {
-        var code = reg.Code.ToByteArray();
-        var output = new Compiler().BuildWasm(code);
-        var executive = new Executive(ExternalEnvironment, output.Contracts.First());
-        return await Task.FromResult(executive);
+        try
+        {
+            var code = reg.Code.ToByteArray();
+            var output = new Compiler().BuildWasm(code);
+            var executive = new Executive(ExternalEnvironment, output.Contracts.First());
+            return await Task.FromResult(executive);
+        }
+        catch (Exception e)
+        {
+            Console.WriteLine(e);
+            throw;
```

Comment:
just wondering: does the caller of this function handle the exception in the end? and for ease of tracing/debug is it better to add a string to `Console.WriteLine(e);` maybe `Console.WriteLine("RunAsync Error:"+e);` or something like that so that we can better trace the error in the logs? or would the stack-trace of the exception make that clear in the logs? thx! 🙇